Source/WebKit/UIProcess/API/APIContentRuleListStore.cpp

```diff
@@ -660,6 +675,38 @@ void ContentRuleListStore::corruptContentRuleListHeader(const WTF::String& ident
     closeFile(file);
 }
 
+void ContentRuleListStore::corruptContentRuleListURLFilter(const WTF::String& identifier)
+{
+    auto path = constructedPath(m_storePath, identifier);
+    if (!FileSystem::makeSafeToUseMemoryMapForPath(path))
+        return;
+    auto fileData = mapFile(path);
+    if (fileData.isNull())
+        return;
+    auto metaData = decodeContentRuleListMetaData(fileData);
+    if (!metaData)
+        return;
+
+    auto file = openFile(constructedPath(m_storePath, identifier), FileOpenMode::ReadWrite);
+    if (file == invalidPlatformFileHandle)
+        return;
+
+    size_t headerAndSourceSize = headerSize(metaData->version) + metaData->sourceSize;
+    size_t actionsOffset = headerAndSourceSize;
+    size_t urlFiltersOffset = actionsOffset + metaData->actionsSize;
+    size_t urlFilterLocationOffset = urlFiltersOffset + 5;
```

Comment:
Could you give 5 a name or calculate it from other values so it's not a magic number?